Mobile/starter_project_dartsmiths/lib/features/login/presentation/widgets/login_card.dart

```diff
@@ -0,0 +1,106 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/src/widgets/framework.dart';
+import 'package:flutter/src/widgets/placeholder.dart';
+
+// ignore: camel_case_types
+class login_card extends StatelessWidget {
+  const login_card({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return Expanded(
+      child: Container(
+        decoration: const BoxDecoration(
+            borderRadius: BorderRadius.only(
+              topLeft: Radius.circular(25),
+              topRight: Radius.circular(25),
+            ),
+            color: Colors.white),
+        child: Padding(
+          padding: const EdgeInsets.fromLTRB(30, 15, 30, 0),
+          child: Column(
+            children: [
+              Column(children: [
+                Row(
+                  children: [
+                    Column(
+                      crossAxisAlignment: CrossAxisAlignment.start,
+                      children: const [
+                        Text(
+                          "Welcome back",
+                          style: TextStyle(
+                              fontWeight: FontWeight.w500, fontSize: 20),
+                        ),
+                        SizedBox(
+                          height: 10,
+                        ),
+                        Text(
+                          "Sign in with your account",
+                          style: TextStyle(fontWeight: FontWeight.bold),
+                        ),
+                      ],
+                    ),
+                  ],
+                ),
+                const SizedBox(
+                  height: 25,
+                ),
+                TextFormField(
+                  decoration: const InputDecoration(
+                      label: Text(
+                        "Username",
+                      ),
+                      hintText: "Jonathandavis@gmail.com"),
+                ),
+                TextFormField(
+                  obscureText: true,
+                  decoration: const InputDecoration(
+                    label: Text(
+                      "Password",
+                    ),
+                    suffixIcon: Text(
+                      "show",
+                      style: TextStyle(
+                        color: Color(0xFF376AED),
+                      ),
+                    ),
+                  ),
+                ),
+                const SizedBox(
+                  height: 50,
+                ),
+                SizedBox(
+                  height: 60,
+                  width: 295,
```

Comment:
The sizes are defined in the Ui converter which is inside the core/utils directory. make use of them.

Reply:
I used UI converter file